pkg/logs/input/file/scanner.go

```diff
@@ -34,6 +35,7 @@ type Scanner struct {
 	registry            auditor.Registry
 	tailerSleepDuration time.Duration
 	stop                chan struct{}
+	sync.Mutex
```

Comment:
if `sync.Mutex` is a composition I think it is better to be on the top of the struct field.

it feel like the mutex is only use to protect the `tailers` map, have you considered to name the mutex according the the map name like `tailersMutex` ?